board/recalbox/fsoverlay/etc/fstab

```diff
@@ -1,12 +1,10 @@
 # /etc/fstab: static file system information.
 #
 # <file system> <mount pt>     <type>	<options>         <dump> <pass>
-/dev/root       /              ext4	ro,noauto         0      1
```

Comment:
noobs sed those two variables.
https://github.com/recalbox/recalbox-rescue/blob/rb-4/sdcontent/os/recalboxOS-rpi3/partition_setup.sh#L16

Did not read the whole pr yet though